paasta_tools/kubernetes_tools.py

```diff
@@ -1487,6 +1549,22 @@ def get_volume_mounts(
                         volume_mounts.remove(existing_mount)
                         break
                 volume_mounts.append(mount)
+
+        if self.config_dict.get("crypto_keys", []):
+            if self.get_crypto_secret_hash():
+                mount = V1VolumeMount(
+                    mount_path="/etc/crypto_keys",
```

Comment:
should we move `"/etc/crypto_keys"` to a named constant? we do refer to it several times in this function already :)